### ISSUE TITLE
concerto-cli compile command not working with target

### ISSUE THREAD

**Issue by j4m3sb0mb** [accordproject/concerto#146]
**Describe the bug**
`concerto-cli compile` command requires `target` as argument but then uses `argv.format` causing this error:
```
Unrecognized target: undefined
```

https://github.com/accordproject/concerto/blob/master/packages/concerto-cli/index.js#L82

**Comment by jeromesimeon** [comment]
> **Describe the bug**
> `concerto-cli compile` command requires `target` as argument but then uses `argv.format` causing this error:
> 
> ```
> Unrecognized target: undefined
> ```
> 
> https://github.com/accordproject/concerto/blob/master/packages/concerto-cli/index.js#L82

Good catch @j4m3sb0mb Thanks. I'll fix this ASAP.

**Comment by jeromesimeon** [comment]
Fixed in #147 Will be in the next release.

Here is a test:
```
bash-3.2$ concerto --version
0.82.3
bash-3.2$ concerto 
10:49:45 - info: Unrecognized target: undefined
bash-3.2$ ~/git/concerto/packages/concerto-cli/index.js compile --target Java --ctoFiles money.cto 
10:49:58 - info: Compiled to Java in './output/'.
```
